docs/framework/whats-new/obsolete-members.md

```diff
@@ -224,14 +224,14 @@ The tables in this article list the type members that are obsolete in the .NET F
 |<xref:System.Security.Permissions.FileIOPermissionAttribute?displayProperty=nameWithType>|<xref:System.Security.Permissions.FileIOPermissionAttribute.All%2A>|Please use the <xref:System.Security.Permissions.FileIOPermissionAttribute.ViewAndModify%2A?displayProperty=nameWithType> property instead.|  
```

Comment:
line 204 should link to System.Security.HostSecurityManagerOptions.HostPolicyLevel

Reply:
Thanks for catching this, @mairaw.